zcash_primitives/CHANGELOG.md

```diff
@@ -6,11 +6,22 @@ and this library adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `zcash_client_backend::AccountBalance::with_unshielded_balance_mut`
 
+### Deprecated
+- `zcash_client_backend::AccountBalance::add_unshielded_value`
+- `zcash_client_backend::AccountBalance::unshielded`
 ## [0.19.0] - 2024-10-02
 
 ### Changed
 - Migrated to `zcash_address 0.6`.
+- Implemented changes to refactor AccountBalance to use Balance for transparent funds
+(see issue #1411). `AccountBalance` now has an `unshielded` value that uses Balance.
+The current implementation maintains retrocompatibility with the `unshielded` value
+represeted with a `NonNegativeAmount`. There are values that are pending to be 
+implemented such as change tracking.
```

Comment:
```suggestion
- Refactored `AccountBalance` to use `Balance` for transparent funds (issue #1411).
  `AccountBalance` now has an `unshielded` value that uses `Balance`. This does not
  currently distinguish between pending change and non-change; the pending value
  is all counted as non-change.
```
Is the intention to fix the latter issue in this PR?

Reply:
Yes! I'm figuring out how to do it 😅. Also I will add more tests.

Reply:
Update: Given how `zcash_client_backend` and the sqlite crate accounts for transparent funds, this is a bigger issue that has dependencies that are out of scope for issue #1411. I created #1592 which describes the follow-up work and (some of) the related issues that are already tracked by the [Zcashd deprecation DAG](https://zcash.github.io/developers/zcash-zcashd-deprecation-dag)